theblues/utils.py

```diff
@@ -15,6 +15,8 @@
 )
 
 
+API_URL = 'https://api.jujucharms.com/'
```

Comment:
I think it would be better and a more minimal change to just set the API_URL to 'https://api.jujucharms.com/v5'